### DISCUSSION TITLE
Json aninhado ( nested objetct ) post request

### DISCUSSION THREAD

**Discussion by rodrigograssi** [balta-io/2811#43] [Q&A] [answered]
Olá, tudo bem?
Estou tentando fazer um post para a minha api com um json aninhado (nested object) mas somente as propriedades primitivas são preenchidas no meu objeto, as propriedades aninhadas não mapeiam, isso é possível?

json body:
{
  "animalId": 1,
  "dadosDoAnimal": {
      "nome": "totó",
      "especie": "canina",
      "raca": "srd",
      "sexo": "F"
    },
}

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @rodrigograssi, o que não está funcionando? Qual erro ocorre?
